2003-09-23  Jeffrey Stedfast  <fejj@ximian.com>

	* camel-mime-utils.c (append_8bit): Don't forget to flush the
	iconv conversion.

Index: camel-mime-utils.c
===================================================================
RCS file: /cvs/gnome/evolution/camel/camel-mime-utils.c,v
retrieving revision 1.183.4.3
diff -u -r1.183.4.3 camel-mime-utils.c
--- camel/camel-mime-utils.c	29 Jul 2003 13:53:29 -0000	1.183.4.3
+++ camel/camel-mime-utils.c	23 Sep 2003 18:37:57 -0000
@@ -1132,7 +1132,9 @@
 		e_iconv_close (ic);
 		return FALSE;
 	}
-
+	
+	e_iconv (ic, NULL, NULL, &outbuf, &outlen);
+	
 	*outbuf = 0;
 	g_string_append(out, outbase);
 	g_free(outbase);
